test/errors/src/index.md

```diff
@@ -19,6 +19,20 @@ Modules = [NonExistantModule]
 NonExistantModule
 ```
 
+```@docs
+# comment in a @docs block
+```
+
+Linking to problematic items: [`ccall`](@ref), [`for`](@ref)
```

Comment:
I think that (in the long run) it would be nice to support keywords without extra syntax. In any case, currently they are creating an error, as they should, so it seems that they are appropriate in the errors section.